libpince/gdb_python_scripts/gdbextensions.py

```diff
@@ -175,14 +175,23 @@ def invoke(self, arg, from_tty):
         if gdbutils.current_arch == typedefs.INFERIOR_ARCH.ARCH_64:
             chunk_size = 8
             int_format = "Q"
-            sp_register = "rsp"
+
+            if arg == "from-base-pointer":
```

Comment:
I believe this is not the best way to check the arguments but it works for now

Reply:
It's okay, several functions use a similar pattern. You could also pass the argument all the time and check if it's false or not to have less lines in the caller